### ISSUE TITLE
Add <o:cdnScript> and <o:cdnStylesheet>

### ISSUE THREAD

**Issue by BalusC** [omnifaces/omnifaces#27]
Since we can control resource rendering via among others the new `Hacks#setScriptResourceRendered()` utility as invented during creating `<o:deferredScript>`, we should be able to use tags like `<o:cdnScript>` and `<o:cdnStylesheet>` to declare CDN resources instead of a relatively clumsy `web.xml` entry as currently done by `CDNResourceHandler`.

**Comment by BalusC** [comment]
I looked at it several times. It's after all not trivial in case we want to maintain the (automatic) resource ordering and also cover resources which are not auto-included by JSF itself.

I concluded it's best to reuse the `CDNResourceHandler` functionality and just use those `<o:cdnXxx>` tags to substitute those context-params. This will only require that `CDNResourceHandler` is by default always enabled.
